ga4gh/cli.py

```diff
@@ -816,3 +818,59 @@ def client_main(parser=None):
     else:
         runner = args.runner(args)
         runner.run()
+
+##############################################################################
+# Configuration testing
+##############################################################################
+
+
+def run_tests(testObj, tests):
```

Comment:
All functions should be camelCase also (the '_main' functions were the only exception to this, which I keep forgetting to fix!). Sorry, should have spotted this earlier.